src/commands/math.ts

```diff
@@ -318,11 +318,18 @@ class MathCommand extends MathElement {
   }
   mathspeakTemplate = [''];
   mathspeak() {
-    // this one runs before a full latex symbol has been formed;
-    //   check if there is more than backslashes in the latex
-    const tex = this.latex().trim();
-    if (this.mathspeakOverride && !/^(\\)+$/.test(tex)) {
-      return this.mathspeakOverride(tex);
+    if (this.mathspeakOverride) {
+      let i = 0;
+      const template = this.textTemplate;
+      var tex = this.foldChildren(template[i], function (fold, block) {
```

Comment:
could `tex` be a const? Also, since `i` will always be 0 at this point, it seems like we might as well inline it:

```suggestion
      const tex = this.foldChildren(template[0], (fold, block) => {
```

Actually, come to think of it... will `template` always be non-empty?

Reply:
I didn't mean to leave a "var" in there-- I was just copying the patterns from around it.